packages/mux-player/src/template.ts

```diff
@@ -1,18 +1,27 @@
-import './media-theme-mux/media-theme-mux';
-import './dialog';
-import { castThemeName, getSrcFromPlaybackId } from './helpers';
-import { html, unsafeStatic } from './html';
+import { document } from 'shared-polyfills';
+import 'media-chrome/dist/media-theme-element.js';
+import './media-chrome/time-display';
 // @ts-ignore
 import cssStr from './styles.css';
+// @ts-ignore
+import muxTheme from './media-theme-mux.html';
+import './dialog';
+import { getSrcFromPlaybackId } from './helpers';
+import { html } from './html';
 import { i18n, stylePropsToString } from './utils';
 
 import type { MuxTemplateProps } from './types';
 import { StreamTypes } from '@mux/playback-core';
 
+const muxTemplate = document.createElement('template');
```

Comment:
question(blocking): Is this needed? From what I can tell, we're

1. pulling in an html template file as a string
2. using JS to create a template element to then set the innerHTML to that string
3. using a clone of that template instance's content as the thing to render into the template.

Couldn't we skip all of that and simply do something like:

```js
const staticTemplateStrPart = `
<style>
  ${cssStr}
</style>
${muxTheme}`;

export const template = (props: MuxTemplateProps) => html`
${staticTemplateStrPart}
${content(props)}
`;
```

? If not, can you briefly explain why not?

Reply:
the html string is automatically escaped so it would just be text. we need to take advantage of the processing of this part https://github.com/muxinc/elements/blob/f24efa9501252d1d7385d359bc06475ecdd7f236/packages/mux-player/src/html.ts#L56-L62

Reply:
ah we should try to resolve that (in a subsequent effort), since, in theory, the `template="templateId"` should do the processing, no?